deepwell-core/src/models/page.rs

```diff
@@ -77,4 +79,17 @@ impl Page {
     pub fn exists(&self) -> bool {
         self.deleted_at.is_none()
     }
+   
+    #[cfg(feature = "ftml_compat")]
+    pub fn create_pageinfo<'a, TScoring: Scoring>(&self, votes: &Votes) -> ftml::PageInfoOwned {
```

Comment:
Why '`TScoring`'? You could just have `S`

Reply:
Mostly a C# convention, where generics are prefixed with a `T` followed by a description of the type. Personally I think it's more explicit, but if you want `S` I can do that.